src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Microsoft/Build/Tasks/Windows/FileClassifier.cs

```diff
@@ -82,7 +82,7 @@ public override bool Execute()
 
             try
             {
-                TaskHelper.DisplayLogo(Log, SR.Get(SRID.FileClassifierTask));
+                TaskHelper.DisplayLogo(Log, "FileClassifier");
```

Comment:
Why are these (and similar) lookups changed from `SR.Get(SRID.SomeName)` to `"SomeName"` ?

Reply:
The generated SRIDs are string variables containing the string of the resource ID (e.g., string FileClassifierTask == "FileClassifierTask").  In this case, the actual resource lookup is the string value "FileClassifierTask" which resolves to the untranslated string "FileClassifier".  For the untranslated case with only a single instance, I've removed the SRID and the resource lookup and replaced it with a hardcoded string.

Reply:
If `"FileClassifier"` continues to reside in `strings.resx`, we should continue using `SR.Get()`. 
If `"FileClassifier"` has been converted into a `const string`, then just use the corresponding variable. 

There is no need to create a shortcut here even for the scenarios that you've described - it is likely to confuse future maintainers of this code (_The generated SRIDs are string variables containing the string of the resource ID (e.g., string FileClassifierTask == "FileClassifierTask")_

The real problem is that we continue to have string resources that are not useful (like `FileClassifierTask`). That should be refactored in a fundamentally different manner (perhaps open an Issue to track it).

Reply:
These can be removed from strings.resx, added to `const string` list, and accessed as before (i.e., `SRID.FileClassifierTask` is replaced by ` const string FileClassifierTask`)